user-guide/Standard_Apps/PA/Getting_Started_with_PA/Creating_and_Configuring_a_Process_Definition.md

```diff
@@ -0,0 +1,132 @@
+
+   Keep this in mind:
+
+   - Always start with a *PA None Event* or *PA Timer Start Event*.
+
+   - Make sure all branches end with a *PA End Event*.
+
+   - For each script task that will be used in a process, make use of the generic *PA Script Task.*
+
+   ![Process Definition](~/user-guide/images/Process_Definition.png)
+
+1. Save the service definition with the *Save All* button.
+
+1. Select each activity individually and give it a meaningful label using the *Edit Label* button in the lower right corner.
+
+   > [!NOTE]
+   > Labels must be unique across the entire process definition.
+
+## Creating and assigning profile instances
+
+1. Make sure there is a profile instance for the PA None Start Event. If this profile instance does not exist yet, create one in the [*Profiles* module](xref:The_Profiles_module) that applies to the PA None Start Event, with the following values:
+
+   - *PA Initial Process Instantiation*: Set this to *Disabled*.
+   - *PA Business Key*: Leave this set to *Not configured*.
+   - *PA DOM Reference*: Leave this set to *Not configured*.
+
+     ![Profile_Instance](~/user-guide/images/Profile_Instance.png)
+
+   Alternatively, start the process with a *PA Timer Start Event* that can be used to generate new tokens periodically (see [Pushing tokens periodically](xref:Pushing_Tokens_Into_A_Process#pushing-tokens-periodically)).
+
+1. In the *Process Automation* app, select the newly created process definition.
+
+1. Click the *Configure* button to open the *process definition configuration* wizard.
+
+1. On the first page of the wizard, select the process DOM definition and click *Next*. A page will be shown where all nodes in the process definition are listed.
+
+1. For each node in the process definition:
+
+   1. Select the node and click *Configure*.
+
+   1. Provide the requested information. This will mainly be related to the profile instance selection, profile instance creation, and the link with DOM field descriptors.
+
+      ![Process_Definition_Configuration](~/user-guide/images/Process_Definition_Configuration.png)
+
+      For example, for a "Ping IP" script task, the following input would be needed in the wizard:
+
+      - **Profile Definition**: The profile definition set up during the creation of the "Ping IP" script task.
+
+      - **Profile Instance**: An existing or new profile instance.
+
+      - **Profile Parameters**: A list of profile parameters that are part of the profile definition.
+
+        Parameters are used to define the input and output of the activity. Input parameters can contain hard-coded values. In that case, the same value will always be used. Alternatively, they can refer to a field descriptor from the process DOM instances. In that case, different values need to be used for each process run.
```

Comment:
different values can be used for each process run ( it depends on what the DOM instance contains )